### ISSUE TITLE
Post Title has character limit, too short

### ISSUE THREAD

**Issue by hansontranhai** [Plaxy-Technologies-Inc/YouPlanets-Bug-Report#32]
![image](https://user-images.githubusercontent.com/77167655/104048236-843edd00-51a8-11eb-8c79-08ad46a2c97e.png)

If we want a limit on character for the post title, maybe give creators more leeway? I can barely write anything in 20 characters.

**Comment by Seraph-YCZhang** [comment]
@JalenYes How long should be a reasonable length limit?

**Comment by hansontranhai** [comment]
Historically, 9% of Tweets in English hit the character limit. This reflects the challenge of fitting a thought into a Tweet. I would use 280 characters like Twitter. More than 100 for sure. You guys can decide.

**Comment by JalenYes** [comment]
> Historically, 9% of Tweets in English hit the character limit. This reflects the challenge of fitting a thought into a Tweet. I would use 280 characters like Twitter. More than 100 for sure. You guys can decide.

I guess that's the character limit of the post body, not the post title.  
I did some research. I found that for title, a normal and optimized post title should have up to 50-65 characters. As the post title will be used as the title tag that appears in the search results, so we will set 65 maximum characters for now. 
For post body, current maximum characters is 1000 characters.
